server/backend/api/app/integration/gc_notify.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import requests
+from api.app.schemas import GCNotifyEmailParam
+from api.config import config
+
+LOGGER = logging.getLogger(__name__)
+
+GC_NOTIFY_EMAIL_BASE_URL = "https://api.notification.canada.ca"
+GC_NOTIFY_EMAIL_TEMPLATE_ID = "cd46fd74-7d79-4576-97ef-8b93297def24"
+
+
+class GCNotifyEmailService:
+    """
+    The class is used for sending email
+    """
+
+    TIMEOUT = (5, 10)  # Timeout (connect, read) in seconds.
+
+    def __init__(self):
+        self.API_KEY = config.get_gc_notify_email_api_key()
+        self.email_template_id = GC_NOTIFY_EMAIL_TEMPLATE_ID
+        self.email_base_url = GC_NOTIFY_EMAIL_BASE_URL
+        self.headers = {
+            "Content-Type": "application/json",
+            "Authorization": "ApiKey-v1 " + self.API_KEY,
+        }
+
+        self.session = requests.Session()
+        self.session.headers.update(self.headers)
+
+    def send_email(self, params: GCNotifyEmailParam):
```

Comment:
I don't see where the content for the email is supplied. Presumably the template_id relates to this somehow.

I also don't see where the from and reply-to email addresses are configured, maybe that's handled by the GC-Notify service.

Normally we'd want all emails to have a common subject prefix and a common body suffix.
E.g. email subject: FAM [ENV] - 
email suffix: This email was sent automatically by the FAM application from an unmonitored email account. Or, provide a valid reply-to if we actually want email interaction with end users (unlikely).

Reply:
The email content and from are all set in GC Notify. It's just a simple message for now:

<img width="1055" alt="image" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/d7cf34de-0d5c-479e-a8d2-76eacbb174ea">
 

And the config settings:
<img width="1138" alt="Screen Shot 2023-08-14 at 3 47 59 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/54db80f1-714d-48ee-858c-0dfc24afd09f">

Reply:
Okay, that's good, we should likely name the method send_granted_access_email